### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KoukiHama**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+# OpenChain SBOM Study Group
+
+Welcome to the GitHub repository of the **OpenChain SBOM Study Group**. This group is dedicated to studying and promoting the use of Software Bill of Materials (SBOM) in compliance with OpenChain standards and best practices.
+
+## Purpose
+The SBOM Study Group focuses on discussions, collaborative efforts, and research to foster a deeper understanding of SBOM and its applications in open source compliance, security, and transparency.
+
+## How to Participate
+We encourage anyone interested in SBOM to join the discussions and activities of the OpenChain SBOM Study Group.
+
```
Comment (NorioKobota) [inline README.md:11]: Could you add a 'Join our Mailing list' chapter?
### Join our Mailing list
Please join our mailing list by sending an email to: [sbom+subscribe@lists.openchainproject.org](mailto:sbom+subscribe@lists.openchainproject.org)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+### Join our Slack Channel
+To participate in our conversations, please join the OpenChain SBOM Study Group on Slack. If you don't have an OpenChain Slack account, you can create one by using the link below:
+
+[Join OpenChain Slack](https://join.slack.com/t/openchainproject/shared_invite/zt-7ayys8g2-dgijHIK_kyrhEWEknrD0cQ)
+
+Once your account is created, you can find us in the **#sbom-sg** channel.
+
+## Resources
+- [SBOM Study Group Kick-off Recording](https://openchainproject.org/news/2024/07/30/sbom-study-group-kick-off-recording)
```
Comment (NorioKobota) [inline README.md:20]: Recording info should be included the meeting notes, so I'd like to remove this line. And please keep the kickoff announcement line for record of the launch this study group.

**Review by NorioKobota — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KoukiHama) [reply README.md:10]: I have addressed the comments.

File: README.md (hunk 2 above)
Comment (KoukiHama) [reply README.md]: I have addressed the comments.

**Comment by KoukiHama** [comment]
@NorioKobota 
I have addressed all the comments.

**Review by NorioKobota — APPROVED** [review]
LGTM

**Pull request merged**